src/components/Forms/Forms.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Icon, IconName } from "../../../../components";
+
+/**
+ * The properties for the forms.
+ */
+
+export type FormsProps = {
+  headline?: string;
+  contentURL?: string;
+  isActive?: boolean;
+  options?: [] => string;
+  imageUpload?: boolean;
+  response?: string;
+  image?: string;
+  link?: string;
+  children?: React.ReactNode;
+  onSubmit?: () => void;
+  onNext?: () => void;
+  onPrevious?: () => void;
+  onChange?: () => void;
+}
+/**
+ * A component for forms.
+ * @param props
+ * @constructor
+ */
+export const Forms = (props: FormsProps) => {
+  const Topic = () => {
```

Comment:
What's our rationale behind nesting components here? Any reason not to make these into exporting components in their own right?